image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -273,26 +276,26 @@ spec:
         [< else ->]
         image: {{ ._rox.scanner.dbImage.fullRef | quote }}
         [< end ->]
+        env:
+          {{- include "srox.envVars" (list . "deployment" "scanner-db" "db") | nindent 10 }}
         ports:
-        - name: https-db
+        - name: tcp-postgresql
+          protocol: TCP
           containerPort: 5432
         resources:
           {{- ._rox.scanner._dbResources | nindent 10 }}
-        env:
-          {{- include "srox.envVars" (list . "deployment" "scanner-db" "db") | nindent 10 }}
-        securityContext:
-          runAsUser: 70
-          runAsGroup: 70
         volumeMounts:
-          - name: db-data
+          - name: scanner-db-data
             mountPath: /var/lib/postgresql/data
           - name: scanner-db-tls-volume
             mountPath: /run/secrets/stackrox.io/certs
-          - name: scanner-db-password
-            mountPath: /run/secrets/stackrox.io/secrets
+            readOnly: true
```

Comment:
Does this being read-only affect us updating the certs for local-scanner, for example? I don't think so because this is really just mounting a k8s secret, but just checking. @SimonBaeumer